data/strings/strings.txt

```diff
@@ -15592,7 +15594,7 @@
 
   [error_enter_correct_instagram_page]
     tags = android
-    en = Enter a valid Instagram web address or account name
+    en = Enter a valid Instagram username or URL
```

Comment:
Non-technical users do not know what URL means.

Reply:
fair, will change back